wallet/src/actors/app/handlers/create_wallet.rs

```diff
@@ -14,6 +13,7 @@ pub struct CreateWalletRequest {
     seed_source: String,
     seed_data: types::Password,
     overwrite: Option<bool>,
+    backup_password: Option<types::Password>,
```

Comment:
I would comment that this password only is required if the seed source is `xprv` and it refers to the key used to encrypt the wallet master key (or in case of `xprvdouble` both external and internal keys)

Reply:
Should we return an error if this field is provided but it is not used?

Reply:
Could make sense in the validation function. :+1: